modules/docs/docgen/docParser.ts

```diff
@@ -1705,7 +1729,7 @@ export function getDefaultFromTags(tags: ts.JSDocTagInfo[]): Value | undefined {
         return {kind: 'primitive', value: text as PrimitiveValue['value']};
       }
       if (['true', 'false'].includes(text)) {
-        return {kind: 'boolean', value: Boolean(text)};
+        return {kind: 'boolean', value: text === 'true' ? true : false};
```

Comment:
I am pretty I am confident this can just be ``text === 'true'` and always evaluate to `true` or `false` as desired.
```suggestion
        return {kind: 'boolean', value: text === 'true'};
```

Reply:
Yes!